tests/baselines/reference/conditionalTypes1.errors.txt

```diff
@@ -50,7 +50,7 @@ tests/cases/conformance/types/conditional/conditionalTypes1.ts(159,5): error TS2
 tests/cases/conformance/types/conditional/conditionalTypes1.ts(160,5): error TS2322: Type 'T' is not assignable to type 'ZeroOf<T>'.
   Type 'string | number' is not assignable to type 'ZeroOf<T>'.
     Type 'string' is not assignable to type 'ZeroOf<T>'.
-tests/cases/conformance/types/conditional/conditionalTypes1.ts(263,9): error TS2403: Subsequent variable declarations must have the same type.  Variable 'z' must be of type 'T1', but here has type 'T2'.
+tests/cases/conformance/types/conditional/conditionalTypes1.ts(263,9): error TS2403: Subsequent variable declarations must have the same type.  Variable 'z' must be of type 'T1', but here has type 'Foo<T & U>'.

```

Comment:
This seems worse, but I believe I understood you to say it's just a return to our old behavior?

Reply:
Yes, it's a return to the old behavior, the rationale being that `Foo<xxx>` is accessible outside the function whereas `T2` is not. I think very little code will be affected by this change, as it is relatively rare to have local type aliases.